test/libsolidity/smtCheckerTests/operators/conditional_assignment_2.sol

```diff
@@ -0,0 +1,17 @@
+pragma experimental SMTChecker;
+
+contract C {
+	function f(uint a, uint b) public pure {
+		require(a > 20);
+		require(a < 100);
+		require(b < 3);
+		uint c = (b > 0) ? (a / b++) : (10 + b++);
```

Comment:
Because of expression like these I added logic with visitBranch.
But, I did something wrong in `visit()` method so solver fails...